cloud-aws-common/build.gradle

```diff
@@ -47,6 +47,8 @@ dependencies {
   api group: 'software.amazon.awssdk', name: 'autoscaling', version: awsSdkVersion
   api group: 'software.amazon.awssdk', name: 'cloudformation', version: awsSdkVersion
 
+  implementation group: 'org.springframework', name: 'spring-aspects', version: springFrameworkVersion
```

Comment:
why is this required for aws-common?

Reply:
Compile/runtime error in the absence of this dependency.